docs/05-staking/03-how_to_stake_on_eon.md

```diff
@@ -0,0 +1,89 @@
+- `blockSignPublicKey`: is the address that will sign the block when forged. It can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used;
+- `vrfPubKey`: this is the key created in the previous step;
+- `value`: the amount of ZEN expressed in Satoshis (10^8) to put on stake, that will represent your voting power.
+
+#### How to create a forging stake using web3js
+Below there is an example (written in javascript to run in Remix IDE) of how to create the forging stake by interacting with the contract:
+```
+(async () => {
+    try {
+
+        console.log('Running testWeb3 - Delegate or create a new Forging Stake script...')
+
+        const smartContractAddress = "0x0000000000000000000022222222222222222222";
+        const amount = 0.001; // amount of ZEN of the new stake
+        const accounts = await web3.eth.getAccounts();
+        console.log('Account ' + accounts[0]);
+
+        const abi = require("browser/contracts/artifacts/abi_fs.json"); // Change this for different path
+
+        const contract = new web3.eth.Contract(abi, smartContractAddress, {from: accounts[0]});
+
+        // Testing method:  delegate
+        const ownerAddress = accounts[0];
+        // pick one of the existing forgers, or create a new forger
+        const blockSignPublicKey = "0x" + YOUR_BLOCK_SIGN; // the address that will sign the block when forged; it can be the same address as the ownerAddress or a different one (delegate forging); if this value is not specified the ownerAddress will be used
+        const forgerVrfPublicKey = "0x" + YOUR_VRF_KEY; // this is the key created in the previous step
+        const first32BytesForgerVrfPublicKey = forgerVrfPublicKey.substring(0, 66);
+        const lastByteForgerVrfPublicKey = "0x" + forgerVrfPublicKey.substring(66, 68);
+
+        methodName = 'delegate';
+        console.log('Response for '+ methodName);
+
+        const response = await contract.methods.delegate(blockSignPublicKey,first32BytesForgerVrfPublicKey,lastByteForgerVrfPublicKey,ownerAddress).send({value: amount *10**18}).then(console.log);
+
+    } catch (e) {
+        console.log("Error:" + e.message);
+    }
+  })()
+```
+⚠️ **Please, remember to replace YOUR_BLOCK_SIGN and YOUR_VRF_KEY with the actual keys, and make sure to specify the path of the native contract ABI (not provided in the documentation, but available here together with the complete Remix workspace)!**  
+⚠️ **Please, notice that with this operation you are increasing the voting power of the forger defined by the blockSignPublicKey and forgerVrfKey, and with that the chance to produce blocks. The operation is a delegation of stake but no rewards will be automatically forwarded to you if you are not the owner of both blockSignPublicKey and forgerVrfKey. Double check them before executing the transaction.
+The transaction is reversible by executing a transaction signed by the ownerAddress.**
```

Comment:
delete ** at the end of sentence

Reply:
I cannot delete ** at the end of the sentence because it's the end of bold format and not a typo: if you want the bold format to be removed we have to remove also the starting ** at line 78